src/app/public/modules/card/card.component.scss

```diff
@@ -14,6 +14,7 @@
     // will attempt to vertically align the cards based on the baselines of the text in each card.
     vertical-align: top;
     width: 350px;
+    position: relative;
```

Comment:
Should we be worried about this change's effects on anything a consumer positions inside the card?

Reply:
We discussed offline and decided to leave this